File: online-judge-solutions/Codeforces/401A.py
#https://codeforces.com/problemset/problem/401/A
#Lang		:	Python 3.8
#Time		:	61 ms
#Memory         :	0 KB
import math
n,x=map(int,input().split())
s=abs(sum([i for i in list(map(int,input().split()))]))
print(math.ceil(s/x))

File: online-judge-solutions/Codeforces/401A.cpp
//https://codeforces.com/problemset/problem/401/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	200 KB
#include<iostream>
#include<math.h>
using namespace std;
int main(){
	double n,x,c;
	while(cin>>n>>x){
		int s=0;
		for(int i=0;i<n;i++){
			cin>>c;
			s+=c;
		}
		s=abs(s);
		cout<<ceil(s/x)<<endl;
	}
	return 0;
}
